neps/nep-0171.md

```diff
@@ -276,6 +276,15 @@ interface NftTransferLog {
     memo?: string
 }
 
+// An event log to capture token metadata updating
+// Arguments
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface NftMetadataUpdateLog {
+    token_ids: string[],
+    memo?: string
```

Comment:
in my opinion this should not be needed - this is already in the transaction, and I don't think memo is something that should be indexed. However since this is in other NEP-171 events, we can keep it here.

Reply:
`memo` is not a necessary part of the transaction. There is even no function associated with this event as there is no meaningful one.